context/app/api/client.py

```diff
@@ -211,7 +211,17 @@ def get_vitessce_conf_cells_and_lifted_uuid(self, entity, marker=None, wrap_erro
                 def get_assaytype(entity):
                     uuid = entity.get('uuid')
                     url = f"{current_app.config['SOFT_ASSAY_ENDPOINT']}/{uuid}"
-                    return requests.get(url, headers=self._get_headers()).json()
+                    headers = self._get_headers()
+                    try:
+                        request = requests.get(url, headers=headers)
```

Comment:
This is a nitpick, but technically, requests.get() returns a requests.Response object.
```suggestion 
                        response = requests.get(url, headers=headers)
```

Reply:
Done - correct/logical naming is important for clarity 😄